packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -263,12 +263,14 @@ export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
         }
 
         const validatedSeriesOptions: any[] = [];
-        options.series?.forEach((seriesOptions, index) => {
+        let index = -1;
+        for (const seriesOptions of options.series ?? []) {
```

Comment:
Was this faster?

Reply:
Yes - lambda functions are much harder for the JIT compiler to handle, so they don't get converted to compiled code anywhere near as quickly as normal functions.

This change was a little more tenuous as this did show up as a hotspot in at least one case, and after this change I didn't see it again - but in these situations its generally preferable to err on the side of not using lambdas.